src/elements/services/stylesheetService/StylesheetService.ts

```diff
@@ -0,0 +1,112 @@
+import { IDesignItem } from "../../item/IDesignItem.js";
+import { IProperty } from "../propertiesService/IProperty.js";
+import { IStylesheetService } from "./IStylesheetService.js";
+
+import * as csstree from 'css-tree';
+import { TypedEvent } from "@node-projects/base-custom-webcomponent";
+
+type RuleWithSpecificity = {
+    rule: csstree.RulePlain,
+    specificity: number,
+}
+
+export class StylesheetService implements IStylesheetService {
+
+    stylesheet: string;
+    stylesheetChanged: TypedEvent<{ stylesheet: string; }> = new TypedEvent<{ stylesheet: string; }>();
+
+    ruleset: csstree.StyleSheetPlain;
+
+    constructor(stylesheet: string) {
```

Comment:
string[] for more stylesheets,
maybe also an object like {name:string, stylesheet:string}[]